lib/locales/es.yml

```diff
@@ -2,3 +2,4 @@ es:
   mumukit:
     memory_exceeded: Limite de memoria excedido. ¿Tu programa está alocando demasiada memoria?
     time_exceeded:  Limite de tiempo de ejecución de %{limit}s excedido. ¿Tu programa tiene recursión o bucle infinito?
+    solution: solucion
```

Comment:
This is a filename key, and it should contain only ascii chars. I have renamed the key, though.